config/config.yml

```diff
@@ -27,7 +27,8 @@ users:
   - username: username  # required if entry provided
     password:           # optional (default: random password of size MAGPIE_PASSWORD_MIN_LENGTH)
     email:              # optional (default: <username>@mail.com)
-    group: <groupname>  # optional (one of 'groups' entry, or plain string to generate it with defaults, else anonymous)
+    group: <group.name> # optional (default: anonymous, otherwise one of 'groups' matched by 'name')
```

Comment:
Well here my previous question wasn't related to "groupname" vs "group.name".
I was wondering about why this "groupname" is specified, but not, for example, "email" or "password" in the previous lines.

Reply:
Ah, I see. It is just to make it more obvious it if provided, it can be a reference value that is equal to a name from the `groups` section, such that more parameters can be found. So, to make illustrate it is more than just a plain string.